src/index.html

```diff
@@ -3,15 +3,157 @@
   <head>
     <meta charset="UTF-8">
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    name="viewport"
+    content="width=device-width,
+    user-scalable=no,
+    initial-scale=1.0,
+    maximum-scale=1.0,
+    minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-block">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text"
+          id="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            id="surname"
            autocomplete="off"
          >
```
1 attr per line + proper indentation